packages/eslint-plugin/docs/rules/naming-convention.md

```diff
@@ -201,7 +201,7 @@ There are two types of selectors, individual selectors, and grouped selectors.
 Individual Selectors match specific, well-defined sets. There is no overlap between each of the individual selectors.
 
 - `accessor` - matches any accessor.
-  - Allowed `modifiers`: `abstract`, `override`, `private`, `protected`, `public`, `requiresQuotes`, `static`.
+  - Allowed `modifiers`: `abstract`, `override`, `private`, `protected`, `public`, `requiresQuotes`, `static`, `autoAccessor`.
```

Comment:
It isn't a new modifier - it's a new selector entirely! So it'll need its own dot point listing the allowed modifiers and types.

Reply:
Hmm, yeah, I think you're right - I disagree with my my earlier review suggestion of just going along with the existing `accessor` one.